.github/actions/action.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Run Tests'
+description: 'Run Tests'
+inputs:
+  repo-name:
+    description: 'Repository to be tested'
+    required: true
```

Comment:
I think we can drop this with my comment below about using the action checkout since it will be running on the repo.

I do think instead though we should have an optional input here for `tests` or `test_selection` or something like that which we will use on the `Run Unit Tests` stage as a positional argument for `stestr run`. That will let users filter the tests to just those that are of interest for the repository under test.